src/utils/runtime/src/lib.rs

```diff
@@ -370,9 +371,8 @@ pub fn init_risingwave_logger(settings: LoggerSettings, registry: prometheus::Re
 
         layers.push(Box::new(MetricsLayer::new(registry).with_filter(filter)));
     }
-
-    tracing_subscriber::registry().with(layers).init();
-
+    LogTracer::init().expect("Failed to set logger");
+    tracing_subscriber::registry().with(layers).set_default();
```

Comment:
Not sure whether can replace `tracing_subscriber::registry().with(layers).init();` with `tracing_subscriber::registry().with(layers).set_default();`🤔

Reply:
What does this change do? 👀

Reply:
Oh I might be wrong. It seems to only affect `fmt` layer.

Reply:
Have we tested that we really need this extra step? Calling `init` on the subscriber should already help us to init the `tracing_log` adaptor.

https://docs.rs/tracing-subscriber/latest/src/tracing_subscriber/util.rs.html#58-74

Reply:
> Calling `init` on the subscriber should already help us to init the `tracing_log` adaptor.

Seems right, we only need to enable `tracing-log` feature.

Reply:
Thanks, I see. `tracing-log` is enabled by default, so we should not do anything here.